server/midddleware/middleware.go

```diff
@@ -118,6 +120,10 @@ func Get(config *config.Config, tenantMgr *metadata.TenantManager, txMgr *transa
 	}
 
 	unaryInterceptors = append(unaryInterceptors, []grpc.UnaryServerInterceptor{
+		//grpctrace.UnaryServerInterceptor(grpctrace.WithServiceName(util.Service)),
+		grpc_logging.UnaryServerInterceptor(grpc_zerolog.InterceptorLogger(sampledTaggedLogger)),
```

Comment:
Shall we add a TODO to hook this up to the span IDs? Perhaps perform the logging in the code that manages the high level request?

Reply:
This is not something I changed, I just moved it in the initialization order. This and logs attached to spans are different.